elastic-stack/README.md

```diff
@@ -6,23 +6,23 @@
 
 ### DESCRIPTION
 
-Create links from SignalFx charts and alerts to any URL-addressable destination such as Kibana.
+Create links from Splunk Infrastructure Monitoring charts and alerts to any URL-addressable destination such as Kibana.
 
 The links appear in the Action menus adjacent to metric property values displayed in dashboards and alerts.
 
 The metric property values are used as query parameter and/or path variable values of the URL link.
 
 ### INSTALLATION
 
-Installation is actually configuring linking from property values on SignalFx charts and alerts to any URL-addressable destination. 
+Installation is actually configuring linking from property values on SignalFx charts and alerts to any URL-addressable destination.
 
 Read about the possible configurations [here](https://docs.signalfx.com/en/latest/dashboards/dashboard-links.html) and priority linking [here](https://docs.signalfx.com/en/latest/managing/data-links.html#local-and-global-data-links).
 
 To read about creating local data links, click [here](https://docs.signalfx.com/en/latest/managing/data-links.html#local-links).
```

Comment:
To learn more about creating local data links, see [Configuring local data links](https://docs.signalfx.com/en/latest/managing/data-links.html#configuring-local-data-links)

Does this feature work with local data links?

**NOTE**: The Splunk Documentation Style Guide specifies that link text for all links should be the same as the heading or sentence that is the link target. In particular, don't use "here" as link text.